packages-web/progress-bar/tests/e2e/specs/onClick.spec.ts

```diff
@@ -16,7 +16,8 @@ describe("Progress Bar on click", () => {
         expect(dialog.getText()).toContain(`you are already at ${value}`);
     });
 
-    it("should call Nanoflow", () => {
+    // eslint-disable-next-line jest/no-disabled-tests
```

Comment:
Can you add `MOZ_HEADLESS=1` to travis yaml?

Reply:
Yes! But you don't think it would be duplicated as we have already set the headless for the browser on wdio.conf.js.

"Alternatively, you can pass the -headless command line argument when starting Firefox."

Ref: https://docs.travis-ci.com/user/gui-and-headless-browsers/

Reply:
Nope, we had to set this setting in Travis UI and without it firefox didn't work.

Reply:
And now i removed that flag :)

Reply:
Thanks for adding!